styles/main.css

```diff
@@ -137,27 +137,30 @@ nav a {
     margin-left: 1em;
     margin-right: 1em;
     display: block;
+    border: 4px solid transparent;

```

Comment:
You need to set the transparent border here so that the border takes its space when the `a` element *isn't* being hovered